src/react-apps/applications/runtime/package.json

```diff
@@ -71,7 +71,7 @@
     "eslint": "5.13.0",
     "eslint-config-airbnb-base": "13.1.0",
     "html-loader": "0.5.5",
-    "html-webpack-plugin": "3.2.0",
+    "html-webpack-plugin": "^3.2.0",
```

Comment:
Will it work without the `^` ? So we have fixed version-numbers.

Reply:
seems like that's only problem on local pc. I am removing these changes